test/es-module/test-esm-loader-not-found.mjs

```diff
@@ -0,0 +1,25 @@
+import '../common/index.mjs';
+import { path } from '../common/fixtures.mjs';
+import { ok } from 'assert';
+import { spawn } from 'child_process';
+import { execPath } from 'process';
+
+const child = spawn(execPath, [
+  '--experimental-loader',
+  'i-dont-exist',
+  path('/print-error-message.js'),
+]);
+
+let stderr = '';
+child.stderr.setEncoding('utf8');
+child.stderr.on('data', (data) => {
+  stderr += data;
+});
+child.on('close', () => {
+  stderr = stderr.toString();
+  ok(stderr.includes(
+    'Error [ERR_MODULE_NOT_FOUND]: Cannot find package \'i-dont-exist\' ' +
+    'imported from'
```

Comment:
I think it might be important to include the file path that follows (we want to ensure the error does actually contain it and that it's correct).

Reply:
There’s no path:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'i-dont-exist' imported from /Users/Geoffrey/Sites/node/
    at new NodeError (node:internal/errors:371:5)
```

Reply:
Oh weird. (Out of scope) i think there should be